src/libelektra/kdb.c

```diff
@@ -235,6 +235,14 @@ KDB * kdbOpen(Key *errorKey)
 		errno = errnosave;
 		return handle;
 	}
+	if(elektraMountGlobals(handle, ksDup(keys), handle->modules, errorKey) == -1)
+	{
+#if DEBUG && VERBOSE
+		printf("Mounting global plugins failed\n");
+#endif
+		ELEKTRA_ADD_WARNING(137, errorKey, "Mounting global plugins failed");
```

Comment:
Is there some additional information somewhere? E.g. what was the current key that caused the problem?